hopp/offshore/fixed_platform.py

```diff
@@ -215,20 +226,20 @@ def detailed_output(self):
 def calc_substructure_mass_and_cost(mass, area, depth, fab_cost=14500., design_cost=4.5e6, sub_cost=3000, pile_cost=0):
     '''
     Platform is substructure and topside combined
-    All funstions are based off NREL's ORBIT (oss_design)
-    default values are specified in ORBIT
+    All functions are based off NREL's ORBIT [1] (oss_design.py)
+    default values are specified in [3], 
     '''
     #Inputs needed
     topside_mass = mass
     topside_fab_cost_rate   =   fab_cost   
     topside_design_cost     =   design_cost
 
-    '''Topside Cost & Mass
+    '''Topside Cost & Mass (repurposed eq. 2.26 from [2])
     Topside Mass is the required Mass the platform will hold
     Topside Cost is a function of topside mass, fab cost and design cost'''
     topside_cost   =   topside_mass   *topside_fab_cost_rate  +topside_design_cost
 
-    '''Substructure
+    '''Substructure (repurposed eq. 2.31-2.33)
```

Comment:
From what reference?

Reply:
It should be from [2]. I'll update the comment.